### ISSUE TITLE
xdg-music should probably be read-only.

### ISSUE THREAD

**Issue by Charadon** [flathub/io.github.pragha_music_player.pragha#18]
As far as I know, I don't think pragha ever actually writes anything into a Library folder, so it should probably be read-only to prevent a sandbox escape.

**Comment by Mailaender** [comment]
I agree, it is a player. If it still works with read-only, then file a patch.
